tailor_distro/manage/pin.py

```diff
@@ -86,4 +86,4 @@ def execute(self, repositories, rosdistro_path, distro):
             # TODO(pbovbel) store name of pinner?
             data.status_description = f"Pinned {age} commits behind {source_branch} on {datetime.datetime.now()}"
 
-        self.write_internal_distro()
+        self.rosdistro_repo.write_internal_distro('Pinning {}'.format('/'.join(repositories)))
```

Comment:
If you move this up a level, you'll end up with a separate commit per repository, which is probably a good thing. Then the message could be something like 'Bumping {repo} from {x} to {y}'